src/main/java/com/sequenceiq/cloudbreak/service/decorator/HostGroupDecorator.java

```diff
@@ -8,13 +8,17 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.RequestMethod;
```

Comment:
Please use a boolean or other solution instead of the RequestMethod class, as this introduces a superfluous dependency  into the service.